culture.md

```diff
@@ -54,9 +54,9 @@ The rule in the Code of Conduct about avoiding discussions about identity labels
 
 There have been recurring discussions about some labels over the lifetime of the slack, particularly when it came to using "bi(sexual/romantic)" vs "pan(sexual/romantic)" as an identity label. Both of these labels may mean different things, and the discussion in the LGBTQ community involving them can be particularly contentious. Our experience with these discussions was that it was very easy to be accusative or dismissive of other (for example, implying or stating that bi-identifying individuals or their label are inherently against trans or non-binary people). These discussions were often more disruptive and stressful than fruitful, and it was clear that sometimes the larger context was intentionally or accidentally ignored (for example, the erasure that both bi and pan people so often face in the queer community).
 
-It is a policy of this community that people's own identities (and corresponding labels) must be respected as presented, and since there did not seem to be a clear way to prevent the cycle of challenging and relitigating the issue, a decision was made to ban that, and any such discussion -- this now includes any discussion related to trans vs trans\*, lesbian vs bi, 'gay' and 'queer' as generics, etc. Note that this only applies for self-identification. Forcibly applying a label that you think is fitting to someone who does not self-identify that way is still considered harassment as per our Code of Conduct.
+It is a policy of this community that people's own identities (and corresponding labels) must be respected as presented, and since there did not seem to be a clear way to prevent the cycle of challenging and relitigating the issue, a decision was made to ban any discussion about someone else's identity. This also includes discussion of other identities as they relate to your own ("I am pan because I am not bi, which means...") -- this policy extends to other identities, such as any discussion related to trans vs trans\*, lesbian vs bi, 'gay' and 'queer' as generics, etc. Note that this only applies for self-identification. Forcibly applying a label that you think is fitting to someone who does not self-identify that way is still considered harassment as per our Code of Conduct. All discussions on gender and sexuality contain the expectation that the participants will stay in their own "lane" as it were, and speak only on their own experiences.
 
```

Comment:
* If possible, I think emphasis on *someone else's identity*  at the end of the first sentence would help to highlight the intended scope of this policy from the start

* "Note that this only applies to self-identification" I think that this sentence could be improved by being explicit: "Note that this policy only applies to discussing the self-identification of others" (assuming that *is* the intended meaning, I'm not entirely sure, but I think that that's what is meant). 

* I would then also and add something along the lines of "For instance, it would ***not*** apply to... <a few examples>", again to help clarify and refine the intended scope

* in `#meta` there was this exchange:

> Q: if I say ‘i am trans and what that means to me is $words’ that is a problem but if i say ‘i am trans and what that means to me is in this tumblr i just wrote’ that’s legit?

> A: correct

That seems like a a very fine distinction. I would not have been able to make that distinction from this document alone. I think this document would be improved by explicitly including it as an example of "allowed" and "not allowed". People interact with and approach information in very different ways, and some people (myself for one) will derive intent and meaning much more quickly by having access to examples that can serve as a mental template.

* The last sentence could be a very nice summarization, could it be moved to it's own highlighted para in the middle, and shortened a bit? e.g.:

----
This the intent of this policy can be summarized by:

***In all discussions on gender and sexuality, participants are expected to speak only on their own personal experiences, and not project on to others.***

----

Summarizations are good, they give people a framework to put details.

* as a purely technical suggestion from my experience from writing lots of documentation: assuming your authoring system is insensitive to line breaks, then having one sentence per line is the mode diff- and comment-friendly style of authoring for collaboration.